events/README.md

```diff
@@ -0,0 +1,39 @@
+# events microservice - Large Events (Knative Proof of Portability)
+
+Add, edit, and fetch events list.
+
+These instructions are specific to this microservice. For general instructions and other information, see [the master README.md](../README.md).
+
+### Installing
+
+How to install and run this microservice for local development.
+
+Each microservice has slightly different dependencies and thus slightly different installation procedures. These instructions are specific to the events microservice, but each other subfolder's README.md has an "Installing" section with more accurate instructions.
+
+Make sure your working directory is this microservice's subfolder.
+
+```sh
+cd events
+```
+
+Set up and activate a virtual environment.
```

Comment:
I'd assume a slightly more fluent user, and write this as:

> Install into a virtualenv:
> 
> ```sh
> cd events
> python3 -m venv venv && . venv/bin/activate
> pip3 install -r requirements.txt
> ```

Reply:
Done https://github.com/knative-portability/large-events/pull/39/commits/5b5015aa9b426548820fc25544b4840097090658